homeassistant/helpers/template.py

```diff
@@ -884,6 +884,24 @@ def ordinal(value):
     )
 
 
+def from_json(value):
+    """Convert a string to a JSON object."""
+    try:
+        return json.loads(value)
+    except (ValueError, TypeError):
+        # If string can't be converted
+        return value
```

Comment:
This should not return back the original value if parsing failed. I think something like `None` would be better.

Reply:
Would it even be appropriate to let any exception get thrown so there would be some "invalid JSON format" error in the HA log?

Reply:
Sure, that would be an option - problem with that would be that the user cannot check if a given string is valid JSON (jinja2 has no exception handling i think). But that's probably a very small use case.

Exception is good 👍 (I guess you could even use the exception that json.loads generates itself)

Reply:
Yeah, that's exactly what I was thinking...I'll just remove the try/catch.